src/pyOpenMS/tests/unittests/test_ProteaseDigestion.py

```
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8  -*-
+
+## ----------------------------------------------------------------------------
+## $Maintainer: $
+## $Authors: Hendrik Weisser, Hannes Roest, Stephan Aiche,
+##           Jeremi Maciejewski $
+## ----------------------------------------------------------------------------
+import pyopenms
+
+@report
```

Ruff (0.12.2)

Undefined name `report`

(F821)